UefiPayloadPkg/Tools/MkFitImage.py

```diff
@@ -61,8 +61,11 @@ def BuildFvImageNode(Fdt, InfoHeader, ParentNode, DataOffset, DataSize, Descript
     Fdt.setprop_u32(ParentNode, 'data-offset', DataOffset)
     Fdt.setprop_str(ParentNode, 'compression', 'none')
     Fdt.setprop_str(ParentNode, 'project', 'tianocore')
-    Fdt.setprop_str(ParentNode, 'arch',Arch)
+    if InfoHeader.Arch is not None:

```

Comment:
Can this or the below ever be None? Also, arch's value is the same, as this is what the caller passed in